tests/FSharp.Compiler.Service.Tests/TypedTreePickleTests.fs

```diff
@@ -0,0 +1,166 @@
+﻿module FSharp.Compiler.Service.Tests.TypedTreePickleTests
```

Comment:
I see this as an exploratory tool.

If the build times of the test project make this exploration slower, I believe (?) the same should be doable as an .fsx as well - referencing the compiler and experimenting there.

(just a proposal - this can temporarily live in the test project as well, but maybe adhocly playing with the inner details like here would have a faster feedback loop when in a script file, scratchpad.fsx)